pkg/machines/components/create-vm-dialog/Password.css

```diff
@@ -0,0 +1,23 @@
+/* user config - password strength */
```

Comment:
We are rather inconsistent with number of spaces in `.css` files. We sometimes use 2, sometime 4 and in the best cases we combine 2 and 4 in the same files :)
But I think in css it makes sense to use 4 and I think we use 4 more than 2 :)
But no need to change it to 4 ;)

Reply:
_Most_ web style guides use 2 spaces for HTML and CSS. That's the typical convention, and it has been for decades now.

We all [should be using tabs anyway, as they're more accessible](https://www.reddit.com/r/javascript/comments/c8drjo/nobody_talks_about_the_real_reason_to_use_tabs/).

Reply:
(But, yeah, right now, we should be using whatever the file uses. Try not to mix-and-match in the same file.)

Reply:
Moved to 4 spaces. Because in every machine css file we use 4 spaces.